app_dart/lib/src/datastore/cocoon_config.dart

```diff
@@ -329,8 +343,46 @@ class Config {
         },
       ];
 
-  Future<GitHub> createGitHubClient() async {
-    final String githubToken = await githubOAuthToken;
+  Future<String> generateJKT() async {
+    final String privateKey = await githubPrivateKey;
+    final String publicKey = await githubPublicKey;
+    final JWTBuilder builder = JWTBuilder();
```

Comment:
same here with JWT.

Reply:
This functions are coming from an external library.